draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1658,44 +1656,52 @@ Content-Type: text/plain
   <x:ref>field-vchar</x:ref>    = <x:ref>VCHAR</x:ref> / <x:ref>obs-text</x:ref>
 </sourcecode>
 <t>
+   A field value does not include leading or trailing whitespace. When a
+   specific version of HTTP allows such whitespace to appear in a message,
+   a field parsing implementation &MUST; exclude such whitespace prior to
```

Comment:
should this be "field processing" instead of "field parsing"?

Reply:
Nope, parsing is used when we are talking about parsing the field, whereas processing would be performing the action associated with the semantics of that field.